src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml

```diff
@@ -9,16 +9,40 @@
 
     <!--  #Design  -->
     <StackPanel Margin="5,0">
```

Comment:
Let's get rid of the `StackPanel` and use only the `Grid` (you need to add 2 more rows).